pom.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>uk.gov.companieshouse</groupId>
+		<artifactId>companies-house-parent</artifactId>
+		<version>1.2.0</version>
+	</parent>
+	<artifactId>bankrupt-officer-search-api</artifactId>
+	<version>unversioned</version>
+	<name>bankrupt-officer-search-api</name>
+	<description>CH API for Bankrupt Officer Search Service</description>
+
+	<properties>
+		<java.version>1.8</java.version>
+		<org.mapstruct.version>1.3.0.Final</org.mapstruct.version>
+		<jacoco-maven-plugin.version>0.7.7.201606060606</jacoco-maven-plugin.version>
+		<junit-jupiter-engine-version>5.2.0</junit-jupiter-engine-version>
+		<mockito-junit-jupiter-version>2.18.0</mockito-junit-jupiter-version>
+		<hamcrest-all-version>1.3</hamcrest-all-version>
+		<maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+		<maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+		<spring-boot-dependencies.version>2.1.15.RELEASE</spring-boot-dependencies.version>
+		<spring-boot-maven-plugin.version>2.1.15.RELEASE</spring-boot-maven-plugin.version>
+		<start-class>uk.gov.companieshouse.bankruptofficersearch.api.BankruptOfficerSearchApiApplication</start-class>
+	</properties>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-dependencies</artifactId>
+				<version>${spring-boot-dependencies.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
```

Comment:
What is this needed for?

Reply:
in most recent Java API repos that have been created in CH we use dependency management. I have no good explanation on why this is needed for. I follow examples of more than 5 repos I checked.